### PR TITLE
Feature/species widget (add data to the store)

### PR DESCRIPTION
**Pull request by weberjavi**
This PR adds species data to the store.
It is the first of a series of PRs regarding the species widget.

On this PR `redux-saga` and `axios` have been added. This two libraries will help us to handle side effects (triggered after some action) and will allow us to cancel requests to the server.

Right now MoL API only allow us to request data for individual species. As we have aggregated grid cells sometimes more than 20 requests are made. Through `redux-saga` we trigger species data fetching once a `SET_GRID_CELL_DATA` action is triggered (cancelling non fulfilled requests in case a new action is triggered, to avoid fetching data from species outside the current grid cell on the view).

### CODE REVIEW

**Comment by weberjavi** [comment]
@aabdaab, @kfilip94 ☝️ ready to review

**Comment by weberjavi** [comment]
For your information I've added `gridLayer.outFields = ["*"]` inside an `onLoad` handler to be able to access grid layer data attributes when reloading the application when in landscape mode (so when sharing the app url). Previously I was exposing the attributes on the `landscape manager` but not sure why it was not working on reloading -.-

File: src/pages/data-globe/data-globe-component.jsx [hunk 1]
```diff
@@ -12,22 +11,14 @@ import ToggleUiWidget from 'components/widgets/toggle-ui-widget';
...
-  <Globe sceneId={SCENE_ID} sceneSettings={sceneSettings} >
+const DataGlobeComponent = ({ sceneLayers, activeLayers, isLandscapeMode, onLoad, isSidebarOpen, isCategoriesBoxesVisible, handleZoomChange, handleLayerToggle, sceneSettings }) => (
+  <Globe sceneId={SCENE_ID} sceneSettings={sceneSettings} onLoad={onLoad}>
     <ArcgisLayerManager activeLayers={activeLayers}/>
     <LandscapeViewManager zoomLevelTrigger={8} onZoomChange={handleZoomChange} isLandscapeMode={isLandscapeMode}/>
     <LocationWidget />
     <ToggleUiWidget />
     <ZoomWidget />
     <EntryBoxes isCategoriesBoxesVisible={isCategoriesBoxesVisible} />
-    <Sidebar isSidebarOpen={isSidebarOpen}>
```
[the remark this answers is not in the document]
Comment (weberjavi) [reply src/pages/data-globe/data-globe-component.jsx]: Yeah, I think I remove it because I was not using it at the moment :)

**Comment by aabdaab** [comment]
Merging it then 🙆

**Pull request merged**